web/views.py

```
@@ -4564,20 +4571,15 @@ def create_donation_payment_intent(request):
-                "clientSecret": intent.client_secret,
-                "donation_id": donation.id,
-            }
-        )
+        return JsonResponse({"clientSecret": intent.client_secret, "donation_id": donation.id})
 
     except Exception as e:
         return JsonResponse({"error": str(e)}, status=400)
 
 
 @login_required
 def create_donation_subscription(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_donation_subscription`

(ANN201)